recipes/picobench/all/conanfile.py

```diff
@@ -0,0 +1,76 @@
+from conans import CMake, ConanFile, tools
+import functools
+import os
+
+required_conan_version = ">=1.33.0"
+
+class PicobenchConan(ConanFile):
+    name = "picobench"
+    description = "A micro microbenchmarking library for C++11 in a single header file"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/iboB/picobench"
+    topics = ("benchmark", "header-only")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "with_cli": [True, False],
+        "bind_to_one_core": [True, False],
+    }
+    default_options = {
+        "with_cli": False,
+        "bind_to_one_core": False,
+    }
+    generators = "cmake"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def export_sources(self):
+        self.copy("CMakeLists.txt")
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            self.copy(patch["patch_file"])
+
+    def validate(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, 11)
+
+    def package_id(self):
+        if not self.options.with_cli:
+            self.info.header_only()
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  destination=self._source_subfolder, strip_root=True)
+
+    def _patch_sources(self):
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            tools.patch(**patch)
+
+    @functools.lru_cache(1)
+    def _configure_cmake(self):
+        cmake = CMake(self)
+        cmake.definitions["PICOBENCH_BUILD_TOOLS"] = self.options.with_cli
+        cmake.definitions["PICOBENCH_BIND_TO_ONE_CORE"] = self.options.bind_to_one_core
```

Comment:
Weird, the option `PICOBENCH_BIND_TO_ONE_CORE` is present in the upstream's cmake, but there is no usage 🤔 

https://github.com/iboB/picobench/search?q=PICOBENCH_BIND_TO_ONE_CORE

It seems be leftover, or a future feature.

Reply:
Thank you for your review.
I missed it.
Now PICOBENCH_BIND_TO_ONE_CORE is deleted from recipe.